boards/arm/nucleo_f103rb/pinmux.c

```diff
@@ -53,6 +55,11 @@ static int pinmux_stm32_init(struct device *port)
 
 	stm32_setup_pins(pinconf, ARRAY_SIZE(pinconf));
 
+	/* Enablio AFIO clock */
+	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_AFIO);
```

Comment:
Is that actually required?
AFIO clock activation  done should already be done inside pinmux/gpio driver.

Reply:
As far as I know it is required.
I2C_1 does not work (and I2C init freezes the system) when line 59 is commented out. 
And I could not find a reference to enabling PERIPH_AFIO anywhere else in the zephyr code code.